app/templates/layouts/base.html

```diff
@@ -21,4 +21,12 @@
             </div>
         {% endif %}
     </body>
-</html>
\ No newline at end of file
+<script>
+        $(document).ready(function(){
+            $('.message.close.icon')
+                .click(function(){
```

Comment:
for readability i would put `.click(function(){` on the same line as the selection `$`